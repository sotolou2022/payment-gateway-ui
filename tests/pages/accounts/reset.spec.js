import { mount, enableAutoUnmount } from "@vue/test-utils";
import { beforeEach, afterEach, expect } from "@jest/globals";
import Component from "~~/accounts/reset";

let wrapper;

beforeEach(() => {
  wrapper = mount(Component, { shallow: true });
});

enableAutoUnmount(afterEach);

describe("Test suite for accounts reset page", () => {
  it("is a vue component", () => expect(wrapper.vm).toBeTruthy());

  it("match snapshot", () => expect(wrapper.html()).toMatchSnapshot());
});
