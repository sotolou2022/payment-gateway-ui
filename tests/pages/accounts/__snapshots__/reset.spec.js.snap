// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for accounts reset page match snapshot 1`] = `
"<div class=\\"col-12 col-md-6\\">
  <div class=\\"card shadow rounded-lg\\">
    <div class=\\"card-header fw-bold text-center\\">Reset your password</div>
    <div class=\\"card-body\\">
      <anonymous-stub></anonymous-stub>
      <anonymous-stub></anonymous-stub>
    </div>
  </div>
</div>"
`;
